style.css

```diff
@@ -48,7 +48,7 @@
   text-align: center;
 }
 
-@media screen and (max-width: 640px){
+@media screen and (max-width: 740px){
```

Comment:
You should use rems instead of px, px won't work properly when you are using browser zoom.